webui/react/package.json

```diff
@@ -16,8 +16,8 @@
     "test:coverage": "CI=true npm test -- --env=jsdom --coverage",
     "eject": "react-scripts eject",
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "storybook": "NODE_PATH=src start-storybook -p 9009 -s public",
-    "storybook:build": "NODE_PATH=src build-storybook -s public -o build-storybook"
+    "storybook": "NODE_PATH=src start-storybook -p 9009",
+    "storybook:build": "NODE_PATH=src build-storybook -o build-storybook"
```

Comment:
NOTE: the `-s` option is [going to be deprecated](https://storybook.js.org/docs/react/configure/images-and-assets#%EF%B8%8F-deprecated-serving-static-files-via-storybook-cli).